electrical/calibration.MD

```diff
@@ -0,0 +1,183 @@
+ 1
+-->
+# Open Source Rover: RoboClaw Calibration
+
+# 1. Calibration
+In order for the motors to all work properly and drive the rover correctly, we must calibrate the motors and encoders. We will use a program called Basic Micro Studio to assist with this. BasicMicro Motion Studio can be downloaded at
+
+* [Motion Studio Download](http://www.basicmicro.com/downloads)
+
+We will now go through and check the encoders and motors one at a time and perform the calibration process on each of them. First, make sure that either the robot is fully hanging such that every drive wheel is not in contact with the ground, or that you elevate each drive wheel as you attempt to spin that wheel.
+
+## 1.1 RoboClaw Setup
+1. Connect your computer (with Basic Micro Studio installed) to the RoboClaw via USB- micro USB cable
+2. Open Basic Micro Studio
+3. If prompted, update the RoboClaw Firmware
+
+![](../images/calibration/ion_sutudio_1.png)
+ **Figure 1: Ion Studio Firmware Update**
+  
+4. Click the ’Connect Selected Unit’ box in the upper left
+5. Verify that all the motor control parameters are roughly nominal:
+
+	* Temperature1: 30
+	* M1/M2 Amps: 0.00
+	* M1/M2 Encoder: 0
+	* M1/M2 Speed: 0
+	* Main Battery: Between 11.5-16.7V
+	* Model: 2x7a
+
+![](../images/calibration/is2.png)
+**Figure 2**
+
+
+As you perform the calibration and testing outlined in this document, make sure to monitor the values above and make sure they are changing in the way that you expect they should. For instance, the temperature going much higher than 35 would indicate that something is going wrong, or if you are pulling a large amount of current you might have a short in the wiring.
+
+6. Under the General Settings Tab, you’ll need to change the following settings for each RoboClaw:
+
+![](../images/calibration/is3.png)
+**Figure 3**
+
+	(a) Setup:
+	
+		* Control Mode: Packet Serial
+		* Multi-Unit: Check Enable Multi-Unit Mode 
+
+	(b) Serial:
+
+		* Set the address to 128,129,130,131,132 for each respective RoboClaw
+			*  (Robo- Claw 1 is address 128, 2 is address 129, etc)
+		* Baudrate: 115200
+			
+	(c) Battery:
+			
+		* Max Main Battery: 17.5V 
+		* Min Main Battery: 11.5V
+		* Max Logic Battery: 5.5V Min Logic Battery: 4.5V
+
+	(d) Motors:
+
+		* We will be updating the motor current parameters after calibration is complete. 
+
+	(e) I/O:
+
+		* Encoder 1 Mode: Quadrature for addresses 128,129,130; Absolute for addresses 131 and 132
+		* Encoder 2 Mode: Quadrature for addresses 128,129,130; Absolute for addresses 131 and 132
+
+After changing these settings, make sure to save them! (Device − > Save Settings )
+
+
+
+
+## 1.2 Drive Motor Calibration
+
+7. Perform the following steps for the RoboClaw addresses 128,129, and 130:
+
+	* Go to the PWM settings tab
+
+
+![](../images/calibration/is4.png)
+ **Figure 4**
+  
+	(a) Under the control pane, slowly move the slider bar up for Motor1.
```

Comment:
i would change these (a), (b) , etc to something like N.N.N.a/b/c just to make it play well with the existing formatting. For some reason, github renders these in a weird way